ecs_pod.go

```diff
@@ -127,7 +127,10 @@ func NewBasicECSPod(opts ...*BasicECSPodOptions) (*BasicECSPod, error) {
 
 // Info returns information about the current state of the pod.
 func (p *BasicECSPod) Info(ctx context.Context) (*ECSPodInfo, error) {
-	return nil, errors.New("TODO: implement")
+	return &ECSPodInfo{
+		Status:    p.status,
+		Resources: p.resources,
+	}, nil
```

Comment:
Should the function just return the ECSPodInfo instead of both info and error if error is always nil? Or is this a placeholder?

Reply:
This is required by the `ECSPod` interface and is for future-proofing, because any API that depends on network calls can and will error. I was planning on writing an implementation that uses `DescribeTasks` to get the actual pod status, which can error.

Reply:
This is implementing the `ECSPod` interface and since the `Info` function signature returns an error, this has to return an error whether it is useful for this particular implementation or not.